vNext/extensions/applicationinsights-analytics-js/src/JavaScriptSDK/ApplicationInsights.ts

```diff
@@ -262,12 +265,14 @@ export class ApplicationInsights implements IAppInsights, ITelemetryPlugin, IApp
      * @param pageViewPerformance
      * @param properties
      */
-    public sendPageViewPerformanceInternal(pageViewPerformance: PageViewPerformance, properties?: { [key: string]: any }) {
-        let telemetryItem = TelemetryItemCreator.create<PageViewPerformance>(pageViewPerformance,
+    public sendPageViewPerformanceInternal(pageViewPerformance: IPageViewPerformanceTelemetryInternal, properties?: { [key: string]: any }, systemProperties?: { [key: string]: any }) {
+        let telemetryItem = TelemetryItemCreator.create<IPageViewPerformanceTelemetryInternal>(
+            pageViewPerformance,
             PageViewPerformance.dataType,
             PageViewPerformance.envelopeType,
             this._logger,
-            properties);
+            properties,
+            systemProperties);
```

Comment:
@hectorhdzg  - do you need part A to be set for this api as well? If there is no requirement, lets keep the extra parameter out.

Reply:
Yes we need to send part A to shared events between 1st and 3rd party analytics plugins, pageView, pageViewPerformance and Exception